x/axelarnet/message_handler.go

```diff
@@ -392,20 +392,23 @@ func extractTokenFromPacketData(ctx sdk.Context, ibcK keeper.IBCKeeper, n types.
 }
 
 // deductFee pays the fee and returns the updated transfer amount with the fee deducted
-func deductFee(ctx sdk.Context, b types.BankKeeper, fee *Fee, token keeper.Coin, msgID string) (keeper.Coin, error) {
+func deductFee(ctx sdk.Context, b types.BankKeeper, fee *Fee, token keeper.Coin, msgID string, sourceChain nexus.ChainName, destinationChain nexus.ChainName) (keeper.Coin, error) {
 	if fee == nil {
 		return token, nil
 	}
 
 	feeAmount := funcs.MustOk(sdk.NewIntFromString(fee.Amount))
 	coin := sdk.NewCoin(funcs.Must(token.GetOriginalDenom()), feeAmount)
 	recipient := funcs.Must(sdk.AccAddressFromBech32(fee.Recipient))
+	destinationChain = nexus.ChainName(strings.ToLower(destinationChain.String()))
```

Comment:
why is this necessary? It's the same type. Is this about casing? If so, can you add a comment?